service_agent/package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "debug": "DEBUG=1 nodemon",
```

Comment:
please also use this start script or change it, as it's confusing to have both `debug` and `start:debug`